src/System.Private.Xml/tests/XmlReader/ReadContentAs/ReadAsBooleanAttributeTests.cs

```diff
@@ -41,7 +41,7 @@ public static void ReadContentAsBooleanAttribute12()
             var reader = Utils.CreateFragmentReader("<Root a=' 1 '/>");
             reader.PositionOnElement("Root");
             reader.MoveToAttribute("a");
-            Assert.Equal(true, reader.ReadContentAs(typeof(bool), null));
+            Assert.Equal(false, reader.ReadContentAs(typeof(bool), null));
```

Comment:
Oops.

Reply:
I am assuming this was a string find/replace? How did that one get missed?

Reply:
It didn't get missed.  It was changed with search/replace, but it shouldn't have been replaced because this isn't strongly typed as a bool, rather as object, so I had to manually revert it, and I mis-typed when doing so.

Reply:
How common is this? Should we be explicit in such cases and change it to:
```C#
Assert.Equal((object)true, ...);
```
?

Reply:
What does the explicit-ness help?

Reply:
> Just avoiding issues with future search-and-replace?

That, and also reading code/PR reviews, it is clearer what overload is being called (and it is clear to the reader that the type is being boxed as an object).

I was mainly asking whether we should set a precedent for adding tests in the future, but if it is just ~10, maybe we should fix it in existing tests too (that is, if you agree there is enough value to do so). When modifying related tests, sometimes copy-paste-modify approach is used, so those folks would benefit from the existing tests already being explicit.

I suppose this would occur when calling any API overload with types that have implicit conversions, but it feels like being explicit follows the spirit of our `var` usage guidelines (optimize for reading).

Reply:
Ok, I added casts.